libs/baseapp/gorm-audit-hooks.go

```diff
@@ -0,0 +1,135 @@
+package baseapp
+
+import (
+	"log/slog"
+
+	"github.com/google/uuid"
+	"github.com/guregu/null"
+	model "github.com/satont/twir/libs/gomodels"
+	"github.com/satont/twir/libs/logger"
+	"gorm.io/gorm"
+)
+
+type gormAuditHooks struct {
+	logger logger.Logger
+}
+
+func (c *gormAuditHooks) create(tx *gorm.DB) {
```

Comment:
I think it's better to return error here, because gorm give us this opportunity. So, we can get better observability and consistency between logs and data.

Reply:
Where did you found, that we can return error here? I don't see that in logs for callbacks.

Reply:
My mistake